cocktails9/cocktails9/API/FilterService.swift

```diff
@@ -0,0 +1,36 @@
+class FilterService {
+    
+    private let networkManager: NetworkManager
+    
+    init(networkManager: NetworkManager = .shared) {
+        self.networkManager = networkManager
+    }
+    
+    // Fetch Categories
+    func fetchCategories() async throws -> [String] {
+        let url = "https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list"
```

Comment:
here for example.
```suggestion
        let url = "\(baseUrl)list.php?c=list"
```

Best practice is to have an Enum of paths with url strings.
```
enum Api: String {
  static let baseUrl = "https://www.thecocktaildb.com/api/json/v1/1/"
  static let categories = "\(baseUrl)list.php?c=list"
}
```

Then use it like so:

```
let url = Api.categories
```